user/homework.go

```diff
@@ -46,7 +48,7 @@ func NewHomework(problems []*api.Problem, c *config.Config) *Homework {
 		hw.Items = append(hw.Items, item)
 	}
 
-	hw.template = fmt.Sprintf("%%%ds %%s\n", hw.maxTitleWidth())
+	hw.template = fmt.Sprintf("%%s%%s %%s\n")
```

Comment:
I agree that it feels more correct, but I couldn't find a way to put the spacing directly into the templating when I wanted to left-align the text.